File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/php/lombervid.php
<?php

declare(strict_types=1);

function fizzbuzz(int $number): string
{
    if ($number % 3 == 0) {
        return ($number % 5 == 0) ? "fizzbuzz" : "fizz";
    }

    return ($number % 5 == 0) ? "buzz" : "{$number}";
}

// Main code
for ($i = 1; $i <= 100; $i++) {
    echo fizzbuzz($i) . "\n";
}

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/rust/lombervid.rs
fn main() {
    for i in 1..=100 {
        if i % 3 == 0 {
            if i % 5 == 0 {
                println!("fizzbuzz");
            } else {
                println!("fizz");
            }
            continue;
        }

        if i % 5 == 0 {
            println!("buzz");
            continue;
        }

        println!("{}", i);
    }
}
